include/seqan3/alignment/configuration/align_config_min_score.hpp

```diff
@@ -31,13 +31,13 @@ namespace seqan3::align_cfg
  *
  * ### Example
  *
- * \include test/snippet/alignment/configuration/align_cfg_max_error_example.cpp
+ * \include test/snippet/alignment/configuration/align_cfg_min_score_example.cpp
  */
-struct max_error : public pipeable_config_element<max_error, uint32_t>
+struct min_score : public pipeable_config_element<min_score, int32_t>
 {
     //!\privatesection
```

Comment:
Add constructors as well as a public member `int32_t score{};` See the description of the ticket which demonstrates the usage as reference:
```cpp
auto && min_score = get<seqan3::align_cfg::min_score>(config);
min_score.score = -5;
```